### PR TITLE
Binder

### PR DESCRIPTION
**Pull request by jhamman**
Add's binder configs. 

A simple app is deployed on launch (available at 0.0.0.0:9000). 

I've updated the demo notebook to point at this app.

### CODE REVIEW

File: xpublish/rest.py [hunk 1]
```diff
@@ -90,6 +90,20 @@ def zmetadata_json(self):
...
+
+        da = self._variables[var].data
+        arr_meta = self.zmetadata["metadata"][f"{var}/{array_meta_key}"]
+
+        data_chunk = get_data_chunk(da, chunk, out_shape=arr_meta["chunks"])
+
+        echunk = _encode_chunk(
+            data_chunk.tobytes(), filters=arr_meta["filters"], compressor=arr_meta["compressor"],
+        )
+        return Response(echunk, media_type="application/octet-stream")
```
Comment (jhamman) [inline xpublish/rest.py]: @andersy005 - I didn't mean to commit these changes here but I'll show them off anyways. This doesn't seem to work on its own but I think its a reasonable start toward making the chunk retrieval async friendly. Thoughts?

Comment (andersy005) [reply xpublish/rest.py]: > This doesn't seem to work on its own 

What are the missing pieces?

Comment (jhamman) [reply xpublish/rest.py]: Just that we aren't managing to retrieve keys asynchronously. If you watch the task stream, we never use two workers at the same time on the server side.

Comment (jhamman) [reply xpublish/rest.py]: see #7

**Pull request merged**